.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -36,8 +36,8 @@ it('disallows login to an invalid user', () => {
   cy.clearCookies();
   cy.visit('/wp-login.php')
   cy.fixture('user').then((user) => {
-    cy.get('#user_login').should('not.be.disabled').type(user.username).should('have.value', user.username);
-    cy.get('#user_pass').should('not.be.disabled').type(user.password).should('have.value', user.password);
+    cy.get('#user_login').type(user.username).should('have.value', user.username);
```

Comment:
I didn't use `contains()` here since we are looking at the value, not the text. 
I need a way to make sure input is captured properly, so I would prefer to leave this assertion to achieve double retryability.

Reply:
If this particular input is error-prone and we should use `have.value` here is OK, but we should add a comment in the tests.